BondageClub/Scripts/Timer.js

```diff
@@ -117,7 +117,7 @@ function TimerPrivateOwnerBeep() {
 			ServerBeep.Message = DialogFindPlayer("BeepFromOwner");
 			LogAdd("OwnerBeepActive", "PrivateRoom");
 			LogAdd("OwnerBeepTimer", "PrivateRoom", CurrentTime + 120000);
-			FriendListBeepLog.push({ MemberName: Player.Owner, ChatRoomName: DialogFindPlayer("YourRoom"), Sent: false, Time: new Date() });
+			FriendListBeepLog.push({ MemberNumber: 0, MemberName: Player.Owner, ChatRoomName: DialogFindPlayer("YourRoom"), Private: false, Sent: false, Time: new Date() });
```

Comment:
`MemberNumber` should be kept as `null`ish value (`undefined`/`null`). There are checks elsewhere in code that rely on this (see ex. `FriendListLoadFriendList`)
```suggestion
			FriendListBeepLog.push({ MemberNumber: null, MemberName: Player.Owner, ChatRoomName: DialogFindPlayer("YourRoom"), Private: false, Sent: false, Time: new Date() });
```